### PR TITLE
add `make run-ha-server-[12]` targets

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
Add two new targets to facilitate running a local, high-availability cluster. This will not work without the enterprise repository and corresponding license. This is also not a full substitute for separate machines, since some resources are naturally still shared.

Feedback welcome! (And thanks @mkraft for making it even possible to do this!)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17680

### CODE REVIEW

**Comment by lieut-data** [comment]
cc @prapti

File: Makefile [hunk 1]
```diff
@@ -466,4 +497,4 @@ endif
 
 ## Help documentatin à la https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' ./Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' ./Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```
Comment (grundleborg) [inline Makefile]: I think this change was already made in another (merged) PR.

Comment (mkraft) [reply Makefile]: Yes it's on master: https://github.com/mattermost/mattermost-server/blob/master/Makefile#L470

**Comment by lieut-data** [comment]
Opting to go the route of vagrant boxes instead: sorry for the churn here!

**Pull request closed without merging**